src/main/arduino

```diff
@@ -0,0 +1,30 @@
+#include <LiquidCrystal.h>
+#define BUFF_LEN 64
+
+LiquidCrystal lcd = LiquidCrystal(5, 6, 7, 8, 9, 10, 11);
+int counter = 0;
+int i;
+char buff[BUFF_LEN + 1];
+
+void setup() {
+  Serial.begin(9600);
+}
+
+void loop(){
+
+    if (Serial.available()>0){
+        char data = Serial.read();
+
+        if(data != '\0'){
```

Comment:
Must add one more condition about buff length.
HINT: `buff` should not receive character greater than `BUFF_LEN`.